contracts/src/v0.1/RequestResponseCoordinator.sol

```diff
@@ -39,6 +39,8 @@ contract RequestResponseCoordinator is
 
     PrepaymentInterface s_prepayment;
 
+    mapping(bytes4 => uint256) private sFulfillFunctionSelector;
```

Comment:
I believe this is not used. Can you please remove it?

Reply:
yes, I'll remove this